company-emoji.el

```diff
@@ -117,9 +117,11 @@
 (require 'company)
 (require 'company-emoji-list)
 
-;; version string
+(defconst company-emoji-version "2.5.0"
+  "Current version of company-emoji.")
 
-(defconst company-emoji-version "2.5.0")
+(defconst emojis (company-emoji-list-create)
```

Comment:
You need to prefix this to prevent namespace collision

https://github.com/bbatsov/emacs-lisp-style-guide#naming